conf/messages.en

```diff
@@ -374,3 +375,15 @@ bankDetails.ossWarning = Any changes you make here will also update the bank det
 bankDetails.ossAndIossWarning = Any changes you make here will also update the bank details in your One Stop Shop and previous Import One Stop Shop registrations.
 bankDetails.oneIossWarning = Any changes you make here will also update the bank details in your previous Import One Stop Shop registration.
 bankDetails.iossWarning = Any changes you make here will also update the bank details in all of your Import One Stop Shop registrations.
+
+applicationComplete.title = You are now registered as an IOSS intermediary
+applicationComplete.heading = You are now registered as an IOSS intermediary
+applicationComplete.reference = Your IOSS intermediary number is <br><span class="govuk-!-font-weight-bold">{0}</span>
+applicationComplete.registration.text = We''ve received your registration for <span class="govuk-!-font-weight-bold">{0}</span>
+applicationComplete.registration.p1 = You can now use this service to add clients to your account, then fill out monthly IOSS VAT returns and pay any VAT due.
+applicationComplete.next = What happens next
+applicationComplete.manageAccount = <a class="govuk-link" href="{0}">Go to your account and manage your clients.</a>
```

Comment:
There's a discrepancy between the prototype design and the screenshot in the ticket.
According to the design, the message should read: "Go to your account to add and manage your clients."

We might to need to confirm it with the Designers 🙂